lib/fastlane/actions/slather.rb

```diff
@@ -32,7 +32,7 @@ def self.run(params)
         command += " --show" if params[:show]
         command += " --source-directory #{params[:source_directory]}" if params[:source_directory]
         command += " --output-directory #{params[:output_directory]}" if params[:output_directory]
-        command += " --ignore #{params[:ignore]}" if params[:ignore]
+        command += " #{params[:ignore].split('|').map { |path| "--ignore \"#{path}\"" }.join(' ')}" if params[:ignore]
```

Comment:
I'll have a look at that, thanks for the tip.
Was confused by `is_string: false` in other options, made me thing it's either a string or a bool.